File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--importing-a-form-region-that-is-designed-in-outlook_1.vb
    Private Sub populateComboBox()
        Dim Application As New Outlook.Application()
        Dim outlookNameSpace As Outlook.NameSpace = _
        Application.GetNamespace("MAPI")
        Dim taskFolder As Outlook.MAPIFolder = _
        outlookNameSpace.GetDefaultFolder _
            (Microsoft.Office.Interop.Outlook.OlDefaultFolders.olFolderTasks)
        Dim taskItems As Outlook.Items = taskFolder.Items
        Dim task As Outlook.TaskItem
        For Each task In taskItems
            If Not (task.Subject Is Nothing) Then
                comboBox1.AddItem(task.Subject, System.Type.Missing)
            End If
        Next task
        comboBox1.Text = comboBox1.GetItem(0)
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--importing-a-form-region-that-is-designed-in-outlook_1.cs
        private void populateComboBox()
        {
            Outlook.Application Application = new Outlook.Application();
            Outlook.NameSpace outlookNameSpace = Application.GetNamespace("MAPI");
            Outlook.MAPIFolder taskFolder = outlookNameSpace.GetDefaultFolder(
                Microsoft.Office.Interop.Outlook.OlDefaultFolders.olFolderTasks);
            Outlook.Items taskItems = taskFolder.Items;
            foreach (Outlook.TaskItem task in taskItems)
            {
                if (task.Subject != null)
                {
                    comboBox1.AddItem(task.Subject, System.Type.Missing);
                }
            }
            comboBox1.Text = comboBox1.GetItem(0);
        }